buildtools/ls.js

```diff
@@ -0,0 +1,62 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 Camptocamp SA
```

Comment:
if the code is from the `ls` package, the copyright and license must be from the project

Reply:
But the project is in copy left: https://github.com/awnist/ls/blob/master/UNLICENSE
And our CI requires to have a copyright header...